index.html

```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
         <li><a href=""><img src="./images/v.png" alt="peace"></a></li>
       </ul>
     </div>
-    </section> 
-    <section class="cards">
-      <div class="card">
-        <img src="./images/1.png" alt="card1">
+  </section>
+  <section class="cards">
+    <div class="card">
+      <img src="./images/11.png" alt="card1">
+      <div>
         <h2 class="text-title">Tonic</h2>
         <div class="experience">
-        <span class="subtitle">CANOPY</span>
-        <ul class="block">
-          <li>&bull; Back End Dev</li>
-          <li>&bull; 2015</li>
-        </ul>
+          <span class="subtitle">CANOPY</span>
+          <ul class="block">
+            <li>&bull; Back End Dev</li>
+            <li>&bull; 2015</li>
+          </ul>
         </div>
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p class="text1 ">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
         <ul class="skills">
           <li>html</li>
           <li>css</li>
           <li>Javascript</li>
         </ul>
         <div class="but"><button class="button">See project</button></div>
       </div>
+    </div>
 
-      <div class="card">
-        <img src="./images/2.png" alt="card2">
-        <h2 class="text-title">Multi-post<br>stories</h2>
+    <div class="card card-reverse">
+      <img src="./images/22.png" alt="card2">
+      <div>
+        <h2 class="text-title">Multi-post stories</h2>
         <div class="experience">
-        <span class="subtitle">CANOPY</span>
-        <ul class="block">
-          <li>&bull; Back End Dev</li>
-          <li>&bull; 2015</li>
-        </ul>
+          <span class="subtitle">FACEBOOK</span>
+          <ul class="block">
+            <li>&bull; Full Stack Dev</li>
+            <li>&bull; 2015</li>
+          </ul>
         </div>
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p class="text1">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
         <ul class="skills">
           <li>html</li>
+          <li>Ruby on rails</li>
           <li>css</li>
           <li>Javascript</li>
         </ul>
         <div class="but"><button class="button">See project</button></div>
       </div>
```

Comment:
- [ ] As required in the previous review [here](https://github.com/Jessekay/My-portfolio/pull/4/files#r1117933857), your website should be responsive, kindly make your website responsive to adapt to the user's device, and it makes it easier to use and navigate. Remember that this leads to a better user experience, which can increase the time users spend on your site, reduce bounce rates, and increase conversion rates. :wink: 
![Screenshot from 2023-02-25 19-38-48](https://user-images.githubusercontent.com/88894525/221371640-65208e18-4900-4575-978f-d8d64cb5244e.png)